File: app/src/main/java/tech/leonam/resultadodocertame/view/InfoTurmaView.java
package tech.leonam.resultadodocertame.view;

import android.graphics.Color;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import tech.leonam.resultadodocertame.R;
import tech.leonam.resultadodocertame.modelView.PegaTurmas;
import tech.leonam.resultadodocertame.view.adapters.ReciclerViewAdapter;

public class InfoTurmaView extends AppCompatActivity {
    private ReciclerViewAdapter adapter;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_info_turma_view);
        getSupportActionBar().hide();
        getWindow().setStatusBarColor(Color.BLACK);
        iniciarReciclagem();
    }

    public void iniciarReciclagem() {
        RecyclerView recyclerView = findViewById(R.id.reciclavel);
        adapter = new ReciclerViewAdapter(this, new PegaTurmas().getTurmas(this));
        var layoutManager = new LinearLayoutManager(this, LinearLayoutManager.VERTICAL, false);
        recyclerView.setHasFixedSize(true);
        recyclerView.setLayoutManager(layoutManager);
        recyclerView.setAdapter(adapter);
    }
}

File: app/src/main/java/tech/leonam/resultadodocertame/view/activitys/InfoTurmaView.kt
package tech.leonam.resultadodocertame.view.activitys

import android.graphics.Color
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import tech.leonam.resultadodocertame.R
import tech.leonam.resultadodocertame.modelView.service.PegaTurmasService
import tech.leonam.resultadodocertame.view.adapters.ReciclerViewAdapter

class InfoTurmaView : AppCompatActivity() {
    private var adapter: ReciclerViewAdapter? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_info_turma_view)
        supportActionBar!!.hide()
        window.statusBarColor = Color.BLACK
        iniciarReciclagem()
    }

    fun iniciarReciclagem() {
        val recyclerView = findViewById<RecyclerView>(R.id.reciclavel)
        adapter = ReciclerViewAdapter(this, PegaTurmasService().getTurmas(this))
        val layoutManager = LinearLayoutManager(this, LinearLayoutManager.VERTICAL, false)
        recyclerView.setHasFixedSize(true)
        recyclerView.layoutManager = layoutManager
        recyclerView.adapter = adapter
    }
}